src/System/Taffybar/HostName.hs

```diff
@@ -0,0 +1,51 @@
+
+module System.Taffybar.HostName
+  ( hostNameNew
+  , defaultHostNameConfig
+  , HostNameConfig(..)
+  ) where
+
+import Data.Monoid ((<>))
+import Graphics.UI.Gtk (Widget, toWidget, widgetShowAll)
+import Network.HostName (getHostName)
+
+import System.Taffybar.Widgets.PollingLabel (pollingLabelNew)
+
+-- | The configuration for the hostname label.
+data HostNameConfig =
+  HostNameConfig
+    { hostNameBackgroundColor :: Maybe String -- ^ Background color. If
```

Comment:
can you put this comment on its own line so that the columns aren't so short.

Use -- | if it makes you happy

Reply:
No problem.  I'll fix this.